### PR TITLE
creates xwiki project

### PR DESCRIPTION
**Pull request by otaviojava**
Ref: https://github.com/platformsh-templates/xwiki/pull/1

### CODE REVIEW

File: templates/xwiki/files/.platform.app.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: xwiki
```
Comment (chadwcarlson) [inline templates/xwiki/files/.platform.app.yaml]:
```suggestion
name: app
```

File: templates/xwiki/files/.platform/routes.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# The routes of the project.
+#
+# Each route describes how an incoming URL is going
+# to be processed by Platform.sh.
+
+"https://{default}/":
+  type: upstream
+  upstream: "xwiki:http"
```
Comment (chadwcarlson) [inline templates/xwiki/files/.platform/routes.yaml]:
```suggestion
  upstream: "app:http"
```

**Comment by chadwcarlson** [comment]
See slack thread for additional discussion on setting plan size.

File: templates/xwiki/files/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+* Java 8
+
+## Customizations
+
+* [`.platform/routes.yaml`](.platform/routes.yaml): Platform.sh allows you to define the [routes](https://docs.platform.sh/configuration/routes.html).
+* [`.platform/services.yaml`](.platform/services.yaml):  Platform.sh allows you to completely define and configure the topology and [services you want to use on your project](https://docs.platform.sh/configuration/services.html).
+* [`.platform.app.yaml`](.platform.app.yaml): You control your application and the way it will be built and deployed on Platform.sh [via a single configuration file](https://docs.platform.sh/configuration/app-containers.html).
+
+> To follow the minimum requirement use at least a Large plan type. https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Performances/#HMemory
```
Comment (otaviojava) [inline templates/xwiki/files/README.md]: WDYT?

File: templates/xwiki/.platform.template.yaml [hunk 4]
```diff
@@ -38,6 +38,13 @@ info:
   notes:
     - heading: "Apps & Services"
       content: "Java 8<br/>"
+## Sets the available plans in the same way:
+plans:
+  # Removes plans from the list and bloceks them from being used for this
+  # project.
+  exclude:
+    - development
```
Comment (chadwcarlson) [inline templates/xwiki/.platform.template.yaml]:
```suggestion
    - medium
```

File: templates/xwiki/.platform.template.yaml [hunk 5]
```diff
@@ -38,6 +38,13 @@ info:
   notes:
     - heading: "Apps & Services"
       content: "Java 8<br/>"
+## Sets the available plans in the same way:
+plans:
+  # Removes plans from the list and bloceks them from being used for this
```
Comment (chadwcarlson) [inline templates/xwiki/.platform.template.yaml]:
```suggestion
  # Removes plans from the list and blocks them from being used for this
```

File: templates/xwiki/files/README.md [hunk 6]
```diff
@@ -22,6 +22,8 @@ XWiki is a free wiki software platform written in Java with a design emphasis on
 
 > To follow the minimum requirement use at least a Large plan type. https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Performances/#HMemory
```
Comment (chadwcarlson) [inline templates/xwiki/files/README.md]:
```suggestion
> **note**
>
> This template must be initialized on a *Large* Platform.sh plan, due to XWiki's [minimum memory requirements](https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Performances/#HMemory).
>
> After the project has been initialized, make sure to [increase the size of your application containers on development environments](https://platform.sh/blog/2019/supersize-app-containers-in-your-dev-environments/) as well.
```

File: templates/xwiki/files/README.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+* [`.platform/services.yaml`](.platform/services.yaml):  Platform.sh allows you to completely define and configure the topology and [services you want to use on your project](https://docs.platform.sh/configuration/services.html).
+* [`.platform.app.yaml`](.platform.app.yaml): You control your application and the way it will be built and deployed on Platform.sh [via a single configuration file](https://docs.platform.sh/configuration/app-containers.html).
+
+> **note**
+>
+> This template must be initialized on a *Large* Platform.sh plan, due to XWiki's [minimum memory requirements](https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Performances/#HMemory).
+>
+> After the project has been initialized, make sure to [increase the size of your application containers on development environments](https://platform.sh/blog/2019/supersize-app-containers-in-your-dev-environments/) as well.
+
+*Make sure you turn on large app env or it won't work on dev envs*
```
Comment (chadwcarlson) [inline templates/xwiki/files/README.md]:
```suggestion

```

**Pull request merged**